### PR TITLE
Spherical Accumulator based normal vector estimators

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

This PR implements an estimator from local digital surface patch to estimate normal vectors from spherical accumulators and voting (Boulc'h & Marlert [SGP 2012]). 

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

**Comment by dcoeurjo** [comment]
(I've tried to optimize the random rotations but I think the result is not correct now.. working on it)

**Comment by dcoeurjo** [comment]
Ok ready for review :)

File: src/DGtal/geometry/surfaces/estimation/estimationFunctors/SphericalHoughNormalVectorEstimator.h [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+        ASSERT( k < myPoints.size());
+        
+        RealPoint v = myPoints[i] - myPoints[j];
+        RealPoint u = myPoints[i] - myPoints[k];
+        RealPoint w = myPoints[j] - myPoints[k];
+        
+        //aspect ratio
+        double a = u.norm() , b = v.norm();
+        double c = w.norm();
+        double s = s = (a+b+c)/2.0;
```
Comment (JacquesOlivierLachaud) [inline src/DGtal/geometry/surfaces/estimation/estimationFunctors/SphericalHoughNormalVectorEstimator.h]: double s = (a+b+c)/2.0;

**Comment by dcoeurjo** [comment]
@JacquesOlivierLachaud , Can I merge this PR?

**Comment by dcoeurjo** [comment]
Doing a self-merge.

**Pull request merged**